pkg/scheduler/objects/allocation_test.go

```diff
@@ -465,3 +465,33 @@ func TestNewAllocFromSI(t *testing.T) {
 	assert.Assert(t, !alloc.IsAllowPreemptSelf(), "alloc should not have allow-preempt-self set")
 	assert.Assert(t, !alloc.IsAllowPreemptOther(), "alloc should not have allow-preempt-other set")
 }
+
+func TestGetUint64Tag(t *testing.T) {
```

Comment:
This whole test belongs in the application_test.go file not in the allocation_test.go file.

Reply:
Good catch @wilfred-s , addressed in latest PR.